src/components/BooksContainer/BooksContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { useBooks } from '../../hooks/books/useBooks/useBooks'
+import { Book } from '../Book/Book'
+import styles from './BooksContainer.module.css'
+
+export const BooksContainer = () => {
+  const { loading, error, books } = useBooks()
+
+  if (loading) return <p>Загрузка...</p>
+
+  if (error) return <p>Ошибка: {error.message}</p>
+
+  return (
+    <section className={styles.container}>
+      <div>
+        <h2 className={styles.headerText}>Все книги</h2>
+        {books.length === 0 ? (
```

Comment:
Ты мог не сравнивать значения с 0

```suggestion
        {!!books.length ? (
```

Reply:
+